app/controllers/events_controller.rb

```diff
@@ -46,7 +46,13 @@ def create
 
   def update
     @event = Event.where(uuid: params[:id]).first
```

Comment:
Do both of these searches get called?

Should the second one get an `if @event.blank?` much like line 58

Reply:
Heyo @jrhoads. Both of these should certainly not get called. The lookup via UUID will get removed. Sorry I should have made this PR a draft. Wanted to see why the builds are failing 😢